### DISCUSSION TITLE
Real time tracing of paths

### DISCUSSION THREAD

**Discussion by itsghaniz** [NVlabs/sionna#611] [General]
I was wondering,as sionna uses optix and the RTX GPUs offer real time raytracing. If I feed in the the positions of a transmitter and receiver, would it be possible that I get the traced paths in realtime?

**Comment by merlinND** [comment]
Hello @itsghaniz,

Although the ray tracing might be very fast, the whole system does much more than ray tracing.
For high-performance radio maps computation, please have a look at the [Instant RM](https://github.com/NVlabs/instant-rm) project.

**Reply by itsghaniz** [reply]
Really appreciate the reply and the instant RM map seems to be really interesting. But regarding the previous question, I am just looking for the paths traced by the scene.trace_paths(). I intend to use the output of this function as an input for the neural networks. 
Thanks in advance.

**Reply by merlinND** [reply]
The performance that you will obtain by calling `scene.trace_paths()` will depend on your hardware, scene, the options you enable, number of paths that you trace, etc, so I think it's best if you try it yourself and measure the latency.
We are working on maximizing the performance of ray tracing operations.
